docs/mkdocs_macros.py

```diff
@@ -2,14 +2,32 @@
 
 from helm.benchmark.run_expander import RUN_EXPANDERS
 from helm.proxy.models import ALL_MODELS
+import yaml
 
 
 def define_env(env):
     @env.macro
     def models_by_organization():
+        with open("src/helm/benchmark/static/schema.yaml", "r") as f:
```

Comment:
Use `read_schema` to use the typed objects, which has the `name_to_model` field already...